### PR TITLE
Get value

### PR DESCRIPTION
**Pull request by reginafcompton**

### CODE REVIEW

**Comment by reginafcompton** [comment]
@jeancochrane - ready for review! 

Also, for the people without definable overlap dates, the charts look a bit odd (lots of "unknowns"). The code seems to be doing what's expected, however...so, I am not sure if exploring another solution is worthwhile here.

screen shot 2017-09-08 at 10 46 19 am

**Comment by jeancochrane** [comment]
Awesome, thank you @reginafcompton! Taking a look now.

File: person/views.py [hunk 1]
```diff
@@ -189,15 +189,21 @@ def get_context_data(self, **kwargs):
                         else:
                             overlap_duration = 'Unknown'
 
-                        info['last_cited'] = repr(commander.object_ref.lastciteddate.get_value().value)
+
```
Comment (jeancochrane) [inline person/views.py]: Oops, some stray whitespace here.

**Comment by reginafcompton** [comment]
Let's order this table by end of overlap, rather than last_cited date, since that attribute is not directly visible in the table.

**Comment by reginafcompton** [comment]
@jeancochrane - new and improved table order!

**Comment by jeancochrane** [comment]
Perfect. Merge away @reginafcompton!

**Pull request merged**